test/clinical-clusters/clinical-clusters-spec.js

```diff
@@ -36,4 +36,74 @@ describe('benchmarks getter functions', () => {
+  });
+
+  it('claims 130 should not have any clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var clusters = data.filter(c => c.measureId === '130' && c.submissionMethod === 'claims' && c.clinicalClusters);
+    assert.equal(0, clusters.length);
+  });
+
+  it('claims Eye Care should have 117 in clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var measureIds = ['012', '014', '019', '140', '141'];
+    measureIds.forEach(measureId => {
+      var clusters = data.filter(c => c.measureId === measureId && c.submissionMethod === 'claims');
+      assert.equal(1, clusters.length);
+      var theCluster = clusters[0];
+      var eyeCare = theCluster.clinicalClusters.find(c => c.name === 'eyeCare');
+      assert.include(eyeCare.measureIds, '117');
+    });
+  });
+
+  it('claims Colonoscopy Care should have 130 in clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var measureIds = ['425'];
+    measureIds.forEach(measureId => {
+      var clusters = data.filter(c => c.measureId === measureId && c.submissionMethod === 'claims');
+      assert.equal(1, clusters.length);
+      var theCluster = clusters[0];
+      var eyeCare = theCluster.clinicalClusters.find(c => c.name === 'colonoscopyCare');
+      assert.include(eyeCare.measureIds, '130');
+    });
+  });
+
+  it('claims Endoscopy and Polyp Surveillance should have 320 in clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var measureIds = ['185'];
+    measureIds.forEach(measureId => {
+      var clusters = data.filter(c => c.measureId === measureId && c.submissionMethod === 'claims');
+      assert.equal(1, clusters.length);
+      var theCluster = clusters[0];
+      var eyeCare = theCluster.clinicalClusters.find(c => c.name === 'endoscopyAndPolypSurveillance');
+      assert.include(eyeCare.measureIds, '320');
+    });
+  });
+
+  it('registry Osteoporosis Care should not have 110 in clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var clusters = data
+      .filter(c => c.submissionMethod === 'registry')
+      .filter(c => c.clinicalClusters && c.clinicalClusters.find(cc => cc.name === 'osteoporosisCare'));
+    clusters.forEach(theCluster => {
+      var osteoporosisCare = theCluster.clinicalClusters.find(c => c.name === 'osteoporosisCare');
+      assert.notInclude(osteoporosisCare.measureIds, '110');
+    });
+  });
+
+  it('registry Heart Failure Care should not have 226 in clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var clusters = data
+      .filter(c => c.submissionMethod === 'registry')
+      .filter(c => c.clinicalClusters && c.clinicalClusters.find(cc => cc.name === 'heartFailureCare'));
+    clusters.forEach(theCluster => {
+      var osteoporosisCare = theCluster.clinicalClusters.find(c => c.name === 'heartFailureCare');
+      assert.notInclude(osteoporosisCare.measureIds, '226');
+    });
+  });
```

Comment:
Yes! just asserting that the cluster generator rightly processes the raw data. I got your point, this can go too lengthy overtime. Like every change of an attribute in a measure can break some or the other thing in downstream.